cmd/ltctl/main.go

```diff
@@ -26,7 +26,22 @@ func RunCreateCmdF(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("failed to create terraform engine: %w", err)
 	}
-	return t.Create(true)
+
+	// If DBDumpURI is not set, we seed data. Otherwise,
+	// we just load the dump.
+	initData := config.DBDumpURI == ""
+	err = t.Create(initData)
+	if err != nil {
+		return fmt.Errorf("failed to create terraform env: %w", err)
+	}
+
+	if !initData {
+		err = t.IngestDump()
+		if err != nil {
+			return fmt.Errorf("failed to create ingest dump: %w", err)
+		}
+	}
+	return nil
```

Comment:
Does this logic take into account whether the DB already had something in it? I'm thinking of the following usecase:
1. I run `deployment create` to deploy the cluster. I had DBDumpURI configured so the dump is ingested.
2. I test something, generate a report, whatever
3. Now I change something in the deployment config (let's say for example that I add another node to the app nodes)
4. I run `deployment create` again so that the app node is created.

With this change, `deployment create` resets the DB as well, right? I guess this is ok for most usecases, but I'm wondering whether we should surface this somehow and make the user confirm that they are ok resetting the DB. Thoughts?

Reply:
Yep, we reset the DB. This is the behavior for comparison tests as well right?

Without resetting, I don't think it's even possible to restore a dump right? Then you have to take care of edge-cases like conflicts in IDs and things like that. Maybe we can give a warning at the start, but it seems pretty normal to me.

Reply:
> This is the behavior for comparison tests as well right?

Yes, it is. But this is a more manual process, and I think it would be somewhat normal for people to play with the cluster specs running `deployment create` repeatedly without necessarily wanting to reset the DB.

I think the proper solution for this would be to have some flag that you can set to enable/disable the DB reset. Which is something we should do in other places of the CLI, but that's not something for this PR of course.

Reply:
Good, that makes